tests/index/test_index_document_meilisearch.py

```diff
@@ -197,6 +197,20 @@ def test_add_documents_csv(empty_index, songs_csv):
     assert index.get_primary_key() == "id"
 
 
+def test_add_documents_csv_with_delimiter(empty_index, songs_csv_custom_separator):
+    """Tests adding new documents to a clean index."""
+    index = empty_index("csv-delimiter")
+    response = index.add_documents_csv(songs_csv_custom_separator, csv_delimiter=";")
+    assert isinstance(response, TaskInfo)
+    assert response.task_uid is not None
```

Comment:
Is this really needed?

Reply:
Not really, but all the other tests are like that, so I find it weird to remove it for this one.